DICOM/src/Microsoft.Health.DeID.SharedLib/EncryptFunction.cs

```diff
@@ -7,16 +7,29 @@
 using System.IO;
 using System.Security.Cryptography;
 using System.Text;
-using Newtonsoft.Json;
+using Microsoft.Health.DeID.SharedLib.Settings;
 
 namespace Microsoft.Health.Dicom.DeID.SharedLib
 {
     public class EncryptFunction
     {
         // AES Initialization Vector length is 16 bytes
         private const int AesIvSize = 16;
+        private readonly byte[] _aesKey;
+        private readonly byte[] _privateKey;
+        private readonly byte[] _publicKey;
 
-        public static byte[] EncryptContentWithAES(string plainText, byte[] key, Encoding encoding = null)
+        public EncryptFunction(EncryptionSetting encryptionSetting = null)
+        {
+            encryptionSetting ??= new EncryptionSetting();
+            encryptionSetting.Validate();
```

Comment:
>Validate [](http://example.com/codeflow?start=30&length=8)

if encryptSetting is a new object, this validate will throw an exception...

Reply:
Fixed it, Thanks.